flowmachine/tests/conftest.py

```diff
@@ -214,6 +215,10 @@ def get(self, key):
     def keys(self):
         return sorted(self._store.keys())
 
+    def flushdb(self):
+        if self.flush:
```

Comment:
What's the purpose of this check (and of the `flush` attribute in general)? I can't see `DummyRedis.flush` being set anywhere, so this check will always succeed?

Reply:
Added a comment, but it is used here: https://github.com/Flowminder/FlowKit/blob/8bc0ebbd901097b66c523a87aa7ed09f0c61dd28/flowmachine/tests/test_cache_utils.py#L459 to simulate a bail-out-because-the-redis-is-being-meddled-with-while-we-try-to-sync scenario.

Reply:
Ah sorry, I missed that one. Yup, makes sense (and thanks for renaming the attribute, makes it clearer to read 👍 ).